### ISSUE TITLE
Unsubscribe URL

### ISSUE THREAD

**Issue by basv** [censusbv/DitsNews#15]
The unsubscribe URL, created @ https://github.com/ditsmedia/DitsNews/blob/master/core/components/ditsnews/model/ditsnews/ditsnews.class.php#L137 contain two &amp; 's. These URL's do work in GMail but in my local email program this will result in & amp;amp; 's. This will result in an error when clicking the unsubscribe link because the URL = page?nwl=8&amp;s=9&amp;c=e08e822b6cc59a1d4cd9bfa4956f33b7

I had to change this line 137 and 138 to just &'s to make it work properly.

```
                    $newhref .= '&s='.$subscriber->get('id');
                    $newhref .= '&c='.$subscriber->get('code');
```

**Comment by onepack** [comment]
Yeah, the unsubscribe url doesn't work right now. I saw the issue with the &amp; in thunderbird as well.
